examples/c++/best-practice-ntt/compile.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Exit immediately on error
+set -e
+
+mkdir -p build/example
+mkdir -p build/icicle
+
+# Configure and build Icicle
+cmake -S ../../../icicle/ -B build/icicle -DCMAKE_BUILD_TYPE=Release -DCURVE=bn254 -DG2=OFF
```

Comment:
you can also add `-MSM=OFF` to exclude MSM compilation, which boost the build time significantly

Reply:
done